src/java/io/bazel/rulesscala/specs2/Specs2RunnerBuilder.scala

```diff
@@ -34,22 +35,14 @@ class Specs2PrefixSuffixTestDiscoveringSuite(suite: Class[Any], runnerBuilder: R
 
   override def getName: String = "Aggregate Specs2 Test Suite"
 
-  override def getDescription: Description = {
-    lazy val emptySuiteDescription = {
-      val description = Description.createSuiteDescription(getName)
-      description.addChild(Description.EMPTY)
-      description
-    }
-    Try(super.getDescription).getOrElse(emptySuiteDescription)
-  }
-
-  override def getChildren: util.List[Runner] =
+  override def getChildren: util.List[Runner] = {
     super.getChildren.asScala
       .collect {
-        case r: FilteredSpecs2ClassRunner if r.matchesFilter => Some(r)
+        case r: FilteredSpecs2ClassRunner if r.matchesFilter => Some(new SafeSpecs2Runner(r))
         case _: FilteredSpecs2ClassRunner => None
-        case other => Some(other)
+        case other => Some(new SafeSpecs2Runner(other).asInstanceOf[Runner])
```

Comment:
Instead of a cast can we do:
```scala
val runner: Runner = new SafeSpecs2Runner(other)
Some(runner)
```
Casts are very brittle with respect to refactoring. Even when safe now, they may not be safe after a refactor.

Reply:
Isn't variance fun!
Good point, I'll fix this.